config/initializers/rack-attack.rb

```diff
@@ -0,0 +1,13 @@
+class Rack::Attack
+  # Use redis for caching
+  Rack::Attack.cache.store = Rack::Attack::StoreProxy::RedisStoreProxy.new(Redis.current)
```

Comment:
Make sure the emails in redis are not available outside of the app. (IE publicly.
)

Reply:
It seems like our redis provider is configured to require authorization.